src/Reflection/SignatureMap/functionMap.php

```diff
@@ -9437,7 +9437,7 @@
 'RedisArray::_hosts' => ['array'],
 'RedisArray::_rehash' => [''],
 'RedisArray::_target' => ['string', 'key'=>'string'],
-'RedisCluster::__construct' => ['void', 'name'=>'string', 'seeds'=>'array', 'timeout='=>'float', 'readTimeout='=>'float', 'persistent='=>'bool|false'],
+'RedisCluster::__construct' => ['void', 'name'=>'string', 'seeds'=>'array', 'timeout='=>'float', 'readTimeout='=>'float', 'persistent='=>'bool|false', 'auth'=>'string|null'],
```

Comment:
It's supposed to be `auth=` because the parameteter is optional.

Reply:
Sorry, missed that